js/utils/dom.js

```diff
@@ -10,6 +10,12 @@ function isScrolledIntoView(el, scrollContainer) {
   return isVisible;
 }
 
+// https://developer.mozilla.org/en-US/docs/Web/API/Node/contains
```

Comment:
I actually added it in oblivious to the fact that jQuery provides this functionality. It's probably not a super high priority thing, but I can remove it at some point. I think I'm only using it in a couple of places.